Assets/_BForBoss/_Core/Scripts/UserInterface/PauseViewBehaviour.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using Sirenix.OdinInspector;
+using UnityEditor;
```

Comment:
`using UnityEditor` will probably crash the game when trying to play a buiild

Reply:
having this in here won't crash it, but call it will.

Reply:
Fixed